Description: Changing Qb -> Webcamoid
Author: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2016-01-04
Index: webcamoid-6.2.0/Qb/commons.pri
===================================================================
--- webcamoid-6.2.0.orig/Qb/commons.pri
+++ webcamoid-6.2.0/Qb/commons.pri
@@ -18,7 +18,7 @@
 # Web-Site: http://github.com/hipersayanX/webcamoid
 
 isEmpty(COMMONS_PRI_INCLUDE) {
-    COMMONS_APPNAME = "Qb"
+    COMMONS_APPNAME = "Webcamoid"
     COMMONS_TARGET = $${COMMONS_APPNAME}
     VER_MAJ = 6
     VER_MIN = 2
