Description: Set -lQb to lower case
Author: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2015-11-14
Index: webcamoid-6.2.0/Qb/Plugins/Quark/Quark.pro
===================================================================
--- webcamoid-6.2.0.orig/Qb/Plugins/Quark/Quark.pro
+++ webcamoid-6.2.0/Qb/Plugins/Quark/Quark.pro
@@ -37,7 +37,7 @@ INCLUDEPATH += \
     include \
     ../../include
 
-!win32: LIBS += -L../../ -lQb
+!win32: LIBS += -L../../ -lqb
 win32: LIBS += -L../../ -lQb$${VER_MAJ}
 
 OTHER_FILES += pspec.json
