Description: Set to lower case -lQb and QbQml dir
Author: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2015-11-16
Index: webcamoid-6.2.0/Qb/QbQml/QbQml.pro
===================================================================
--- webcamoid-6.2.0.orig/Qb/QbQml/QbQml.pro
+++ webcamoid-6.2.0/Qb/QbQml/QbQml.pro
@@ -49,7 +49,7 @@ INCLUDEPATH += \
     include \
     ../include
 
-!win32: LIBS += -L../ -lQb
+!win32: LIBS += -L../ -lqb
 win32: LIBS += -L../ -lQb$${VER_MAJ}
 
 DISTFILES = qmldir
@@ -58,7 +58,7 @@ INSTALLS += \
     target \
     qmldir
 
-unix: installPath = $$[QT_INSTALL_QML]/QbQml
+unix: installPath = $$[QT_INSTALL_QML]/qbqml
 !unix: installPath = $${PREFIX}/qml/QbQml
 
 target.path = $$installPath
